packages/application-components/src/components/internals/for-docs/example-wrapper/example-wrapper.js

```diff
@@ -1,43 +1,199 @@
-    }}
-  >
-    <IntlProvider locale="en">
-      <div
-        style={{
-          position: 'relative',
-          width: '100%',
-          height: '400px',
-          border: `1px solid ${customProperties.colorNeutral95}`,
-        }}
-      >
-        <div id={props.containerId} style={{ flex: 1 }} />
-        <ModalController
-          title={props.controllerTitle}
-          buttonLabel={props.controllerButtonLabel}
-        >
-          {props.children}
-        </ModalController>
-      </div>
-    </IntlProvider>
-  </ThemeProvider>
-);
+const PreviewContainer = styled.div`
+  position: relative;
+  width: 100%;
+  height: ${props => props.height};
+  border: 1px solid ${customProperties.colorNeutral95};
+`;
+const ColumnsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+  > * + * {
+    margin: 0 0 0 ${customProperties.spacingM};
+  }
+`;
+const ColumnLeft = styled.div`
+  flex: 2;
+  height: 100%;
+`;
+const ColumnRight = styled.div`
+  flex: 1;
+  height: 100%;
+  overflow: auto;
+`;
+
+const KnobsController = props => {
+  const initialValues = props.knobs.reduce(
+    (mapped, knobConfig) => ({
+      ...mapped,
+      [knobConfig.name]: knobConfig.initialValue,
+    }),
+    {}
+  );
+  return (
+    <Formik
+      initialValues={initialValues}
+      render={formikProps => {
+        const form = (
+          <Spacings.Stack size="m">
+            {props.knobs.map(knobConfig => {
+              switch (knobConfig.kind) {
+                case 'text':
+                  return (
+                    <TextField
+                      key={knobConfig.name}
+                      name={knobConfig.name}
+                      title={knobConfig.label}
+                      value={formikProps.values[knobConfig.name]}
+                      errors={formikProps.errors[knobConfig.name]}
+                      touched={formikProps.touched[knobConfig.name]}
+                      onChange={formikProps.handleChange}
+                      onBlur={formikProps.handleBlur}
+                      onFocus={formikProps.handleFocus}
+                    />
+                  );
+                case 'text-multi':
+                  return (
+                    <MultilineTextField
+                      key={knobConfig.name}
+                      name={knobConfig.name}
+                      title={knobConfig.label}
+                      value={formikProps.values[knobConfig.name]}
+                      errors={formikProps.errors[knobConfig.name]}
+                      touched={formikProps.touched[knobConfig.name]}
+                      onChange={formikProps.handleChange}
+                      onBlur={formikProps.handleBlur}
+                      onFocus={formikProps.handleFocus}
+                    />
+                  );
+                case 'select':
+                  return (
+                    <SelectField
+                      key={knobConfig.name}
```

Comment:
![nottether](https://user-images.githubusercontent.com/2425013/59328237-2e643380-8cec-11e9-8d1d-56e61de99e45.gif)

try tethering it?